projects/templatetags/list_tools.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def index(List, i):
```

Comment:
I guess the name `List` is used here to not overwrite the built-in `list`? `list_object` would also work. :thinking:

Reply:
Yeah, this is a mistake from my side, should not be `List` for sure :)